src/simulators/statevector/qubitvector.hpp

```diff
@@ -1792,6 +1811,21 @@ double QubitVector<data_t>::expval_pauli(const reg_t &qubits,
       phase = std::complex<data_t>(0, 1);
       break;
   }
+}
+
+template <typename data_t>
+double QubitVector<data_t>::expval_pauli(const reg_t &qubits,
+                                         const std::string &pauli) const {
+
+  uint_t x_mask, z_mask, num_y, x_max;
+  std::tie(x_mask, z_mask, num_y, x_max) = pauli_masks_and_phase(qubits, pauli);
+
+  // Special case for only I Paulis
+  if (x_mask + z_mask == 0) {
+    return norm();
+  }
+  std::complex<data_t> phase;
+  compute_phase(num_y, phase);
```

Comment:
With above comment this would be
```suggestion
  auto phase = compute_phase(num_y);
```